services/server/port/main.go

```diff
@@ -104,6 +112,37 @@ func main() {
 	queryCache := viper.GetStringMapString("query_cache")
 	store.InitStore(queryCache)
 
+	// generate server priv key if it does not exist
+	err := generateAndSaveNostrPrivateKey()
```

Comment:
There's a function generateKey() in host.go, probably should reuse that if possible.

Reply:
Grandmaster stuff. Colby knew the code is due for refactoring but we hadn't had time for it yet. These are all good points. I will probably push small changes from tomorrow afternoon to make the next review quicker.